prosopopee/themes/material/static/css/styles.css

```diff
@@ -62,8 +62,8 @@ main {
 }
 
 .bordered-picture .caption__overlay {
-    margin-left: 11.5%;
-    margin-right: 11.5%;
+    margin-left: 11.5% !important;
+    margin-right: 11.5% !important;
```

Comment:
"God css practice" often tell that "!important" is dirty. Is it really needed here?